src/ragas/metrics/critique.py

```diff
@@ -81,6 +83,14 @@ def __post_init__(self: t.Self):
         self.strictness = (
             self.strictness if self.strictness % 2 != 0 else self.strictness + 1
         )
+        self.critic_prompt = CRITIQUE_PROMPT
```

Comment:
self.critic_prompt is not defined in the class above right? 
please fix docs string as well

same for all metrics